src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -125,10 +127,33 @@ std::map<std::string, ngraph::OpSet> Extension::getOpSets() {
         return opset;
     };
 
+    auto snippets_opset = []() {
+        ngraph::OpSet opset;
+
+#define NGRAPH_OP(NAME, NAMESPACE) opset.insert<NAMESPACE::NAME>();
+        NGRAPH_OP(BroadcastLoad, ngraph::snippets::op)
+        NGRAPH_OP(BroadcastMove, ngraph::snippets::op)
+        NGRAPH_OP(ConvertSaturation, ngraph::snippets::op)
+        NGRAPH_OP(ConvertTruncation, ngraph::snippets::op)
+        NGRAPH_OP(Kernel, ngraph::snippets::op)
+        NGRAPH_OP(Load, ngraph::snippets::op)
+        NGRAPH_OP(Nop, ngraph::snippets::op)
+        NGRAPH_OP(PowerStatic, ngraph::snippets::op)
+        NGRAPH_OP(Scalar, ngraph::snippets::op)
+        NGRAPH_OP(Store, ngraph::snippets::op)
+        NGRAPH_OP(Subgraph, ngraph::snippets::op)
+        NGRAPH_OP(Tile, ngraph::snippets::op)
+        NGRAPH_OP(TileScheduler, ngraph::snippets::op)
+#undef NGRAPH_OP
```

Comment:
@riverlijunjie @ilyachur should we use CC-factory here? Looks like this code will use all nodes, even they are not used.

I suppose we need at least to replicate this solution https://github.com/openvinotoolkit/openvino/pull/13189/files or even better - make OpSet to be CC-able out of box. Is the second one possible?

Reply:
I think so, we should enable CC for all `opset.insert()`, but there are other more code blocks need to be done in this file. So I suggest creating another PR to handle it.

Reply:
The ticket 98029 has been created